src/opendr/simulation/human_model_generation/utilities/visualizer.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 1996-2020 OpenDR European Project
```

Comment:
This notice seems wrong. It should probably be 2020-2021

Reply:
I modified it. This mistake seems to be also in face_recognition_learner.py in the master branch.